libft/sources/strings/strings_internal.h

```diff
@@ -0,0 +1,38 @@
+#ifndef STRINGS_INTERNAL_H
+#define STRINGS_INTERNAL_H
+
+# include <stddef.h>
+# include <stdlib.h>
+# include "strings.h"
+# include "../memory/memory.h"
+
+#define MAX_OP 10
+
+typedef struct s_delim
+{
+	char *delim;
+} t_delim;
+
+typedef struct s_delims
+{
+	t_delim		*delims;
+	size_t		count;
+} t_delims;
+
+typedef struct s_string
+{
+	char	*str;
+} t_string;
+
+typedef struct s_string_array
+{
+	t_string	*strs;
+	size_t	count;
+} t_strings;
+
+
+char	**ft_split_buf(const char *s, void *delim_arr, int *sz);
```

Comment:
Isn't delim_arr also const?

Reply:
well, i cast it to the t_delims * nside of the function. if it's ok, i can make prototype with const void *